tests/integration/test_packages.py

```diff
@@ -57,16 +62,17 @@ def test_packages(env_package, env_name, test_env, tmpdir):
     expected_files = env_data["expected_files"]
     utils.assert_expected_files(source_path, expected_files, tmpdir)
 
-    purl = env_data.get("purl", "")
-    deps_purls = []
-    source_purls = []
-    if "dep_purls" in env_data:
-        deps_purls = [{"purl": x} for x in env_data["dep_purls"]]
-    if "source_purls" in env_data:
-        source_purls = [{"purl": x} for x in env_data["source_purls"]]
-
-    if purl:
-        image_contents = [{"dependencies": deps_purls, "purl": purl, "sources": source_purls}]
-    else:
-        image_contents = env_data["image_contents"]
+    image_contents = []
+    for pkg in env_data.get("content_manifest"):
+        purl = pkg.get("purl", "")
+        deps_purls = []
```

Comment:
Nit Pick: Should we name this variable as `dep_purls` instead to be consistent with naming?
```suggestion
        dep_purls = []
```

Reply:
@nirzari Thanks!